homeassistant/components/modbus/base_platform.py

```diff
@@ -91,6 +92,7 @@ def __init__(self, hub: ModbusHub, entry: dict[str, Any]) -> None:
         self._attr_unit_of_measurement = None
         self._lazy_error_count = entry[CONF_LAZY_ERROR]
         self._lazy_errors = self._lazy_error_count
+        self._min_value = entry.get(CONF_MIN_VALUE)
```

Comment:
Why use .get() when you know the entry exist. see 2 lines above.

Reply:
self._min_value = entry.get(CONF_MIN_VALUE) ... not sure what you mean ... this is the only reference to this here?

Reply:
```suggestion
        self._min_value = entry[CONF_MIN_VALUE]
```

Reply:
That would throw in most cases as min value is only on the schema for sensors and this is the base?

Reply:
But in that case, the .get should specify the default value.

Reply:
removed default in __init__.py